src/Microsoft.Health.Fhir.Shared.Core/Features/Conformance/ConformanceProvider.cs

```diff
@@ -68,7 +71,7 @@ public override async Task<CapabilityStatement> GetCapabilityStatementAsync(Canc
                 }
             }
 
-            return _capabilityStatement;
+            return _capabilityStatement.ToTypedElement().ToResourceElement();
```

Comment:
ToResourceElement directly on the poco allows caches the instance

```suggestion
            return _capabilityStatement.ToResourceElement();
```